nomad-gcp/main.tf

```
@@ -123,6 +123,7 @@ data "google_compute_image" "ubuntu_2004" {
 resource "google_compute_firewall" "default" {
   name    = "allow-retry-with-ssh-circleci-server-${var.name}"
   network = var.network
+  project = length(regexall("projects/([^|]*)/regions", var.subnetwork)) > 0 ? regex("projects/([^|]*)/regions", var.subnetwork) : null
```

I think we want `[^/]+`

```suggestion
  project = length(regexall("projects/([^/]+)/regions", var.subnetwork)) > 0 ? regex("projects/([^/]+)/regions", var.subnetwork) : null
```

---

actually my big issue now is `var.subnetwork is a string, known only after apply`
this may not work the way we want it to. 

also totally forgot there's a PR. sorry for spamming your inboxes while i test -_-